airbyte-integrations/connectors/destination-s3/src/test/resources/parquet/json_schema_converter/get_avro_schema.json

```diff
@@ -253,5 +253,44 @@
         }
       ]
     }
+  },
+  {
+    "schemaName": "combined_restriction",
+    "namespace": "namespace8",
+    "appendAirbyteFields": false,
+    "jsonSchema": {
+      "properties": {
+        "created_at": {
```

Comment:
can we also test `anyOf` on array and objects?

Reply:
Good point.

Reply:
The new test case actually revealed a bug. Thanks for the suggestion.